em/emFluxCorrection.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include <iostream>
+#include <cstdlib>
+
+#include "crv.h"
+#include "crvShape.h"
+#include "apfElement.h"
+
+#include "em.h"
+using namespace std;
```

Comment:
This seems to not be used. Should be removed then.

Reply:
Removed.